sdk/python/feast/permissions/server/grpc.py

```diff
@@ -22,11 +21,13 @@ def intercept_service(self, continuation, handler_call_details):
                 metadata=dict(handler_call_details.invocation_metadata)
             )
 
-            print(f"Fetching user for token: {len(access_token)}")
+            logger.info(
+                f"Fetching user details for token of length: {len(access_token)}"
```

Comment:
my 2 cents - may be debug log here is more appropriate.

Reply:
set as debug